src/components/tvShow/tvShowInfo.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TVShowInfo = ({ show, isFavourite, onClick }) => {
+  const favImage = isFavourite ? 'star-filled' : 'star-unfilled';
+  const favAlt = isFavourite ? 'Filled star' : 'Unfilled star';
```

Comment:
Ten alt na ikonce jest trochę bez sensu :D albo nazwij go `Favourited/Not favourited` albo wyrzuć.